backend/.nycrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "all": true,
+  "include": [
+    "src/**/*.js"
+  ],
+  "exclude": [
+    "**/*.integration.js",
+    "**/*.unit.js"
+  ]
```

Comment:
exclude `node_modules` as well (probably does it automatically but better to be explicit)